### PR TITLE
Second

### PR DESCRIPTION
**Pull request by Vladislav-Alex**

### CODE REVIEW

File: modules/0_git/test/main.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
```
Comment (allnes) [inline modules/0_git/test/main.cpp:10]: please use new branch or remove old task

File: modules/2_mnist/include/mnist.hpp [hunk 2]
```diff
@@ -8,13 +8,13 @@
 // [inp] filepath - path to images binary file
 // [out] images   - vector of images
 void loadImages(const std::string& filepath,
-                std::vector<cv::Mat>& images);
```
Comment (allnes) [inline modules/2_mnist/include/mnist.hpp:11]: please remove redundant spaces

File: modules/2_mnist/src/mnist.cpp [hunk 3]
```diff
@@ -6,14 +6,13 @@ using namespace cv;
 inline int readInt(std::ifstream& ifs) {
     int val;
     ifs.read((char*)&val, 4);
-    // Integers in file are high endian which requires swap
```
Comment (allnes) [inline modules/2_mnist/src/mnist.cpp:9]: please remove old tasks

**Comment by dkurt** [comment]
see https://github.com/dkurt/openvino_practice/pull/193

**Pull request closed without merging**